src/main/java/frc/robot/subsystems/pivot/PivotConstants.java

```diff
@@ -4,7 +4,7 @@ public class PivotConstants {
   public static final double PIVOT_MASS = 0.0;
   public static final double PIVOT_LENGTH = 0.0;
   public static final double PIVOT_GEARING = 0.0;
-
+  public static final double RIO_FREQUENCY = 0.02;
```

Comment:
This value is in seconds, so frequency may be a bit confusing

Reply:
This should be in HardwareConstants.

Or you can create a new SimConstants class if u wants idk

Reply:
i added it to hardware constants